.github/workflows/lint-test.yml

```diff
@@ -32,6 +32,25 @@ jobs:
     - name: Run pre-commit
       uses: pre-commit/action@v2.0.3
 
+  docs:
+    # We want to run on external PRs, but not on our own internal PRs as they'll be run
+    # by the push to the branch. Without this if check, checks are duplicated since
+    # internal PRs match both the push and pull_request events.
+    if:
+      github.event_name == 'push' || github.event.pull_request.head.repo.full_name !=
+      github.repository
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up environment
+      uses: ./.github/actions/setup-env
+      with:
+        python-version: 3.8
+        nox-setup: "docs"
+
+    - name: Run sphinx-build
+      run: nox -s docs -- -W -w $GITHUB_STEP_SUMMARY
```

Comment:
```suggestion
      run: nox -s docs -- --keep-going -W -w $GITHUB_STEP_SUMMARY
```

Reply:
including keep-going makes sphinx build the docs and exit with 0, which means that this will always pass.